compiler/rustc_codegen_ssa/src/mir/debuginfo.rs

```diff
@@ -36,10 +34,6 @@ pub struct PerLocalVarDebugInfo<'tcx, D> {
     /// `DIVariable` returned by `create_dbg_var`.
     pub dbg_var: Option<D>,
 
-    /// Byte range in the `dbg_var` covered by this fragment,
-    /// if this is a fragment of a composite `VarDebugInfo`.
-    pub fragment: Option<Range<Size>>,
```

Comment:
I think this field actually needs to be used to produce correct debuginfo for composite debuginfo variables.